R/pivot_wider_transition_risk_profile.R

```diff
@@ -0,0 +1,212 @@
+#' Calculate the indicator "transition risk profile"
+#'
+#' @param pivot_wider Logical. Pivot the output at company level to a wide
+#'   format?
+#'
+#' @return A data frame with the column `companies_id`, and the nested
+#'   columns`product` and `company` holding the outputs at product and company
+#'   level.
+#' @export
+#' @keywords internal
+#'
+#' @examples
+#' \dontrun{
```

Comment:
@maurolepore I used `\dontrun` to avoid running un-exported functions. The examples use internal functions like `add_transition_risk_category_at_company_level()` which are not exported internally. Should I export these functions internally to use them in the examples? or usage of `\dontrun` seems more logical here?

Reply:
OK thanks for explaining. 

There is a tension between documenting and maintaining internal code. Documentation clarifies usage but also adds maintenance cost. 

It's specially important to minimize the non-executable references to internal code -- such as comments, or tests and examples that don't actually run. Internal code that won't run also won't ever fail.  That's bad because it'll become outdated and confusing.

Internal functions exported with `keywords @internal` suffer from this problem so it's best to keep them to a minimum.